KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -0,0 +1,81 @@
+# [kubevirtci](README.md): Testing kubevirtci with locally provisioned cluster
+
+After making changes to a kubevirtci provider, you should test it locally before publishing it.
+
+Let's go on the required steps, starting by provisioning, all the way to `make cluster-up`.
+`$KUBEVIRTCI_DIR` is assumed to be your kubevirtci path.
+
+Steps:
+
+### kubevirtci: provision cluster locally
+
+```bash
+# switch to kubevirtci directory
+cd $KUBEVIRTCI_DIR
+```
+
+```bash
+# Build a provider. This includes starting it with cluster-up for verification and shutting it down for cleanup.
+(cd cluster-provision/k8s/1.19; ../provision.sh)
+```
+
+### prepare for using the new provisioned cluster
+
+```bash
+# set local provision test flag (mandatory)
+export KUBEVIRTCI_PROVISION_CHECK=1
+```
+
+Note:
+In case you updated gocli and need to test it locally as well, export additionally:
+```bash
+export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest                                     
```

Comment:
```suggestion
export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
```
I cannot stand trailing whitespace

Reply:
Removed most, forgot 2
Done

Best to let sed do that xD